project-setup/communication-tooling.md

```diff
@@ -0,0 +1,84 @@
+# Title
+
+Communication tooling
+
+# Context
+
+A team depends on another team's component. It would like to make contributions
+to that component. Even when it happens in writing, communication happens in a
+1-on-1 fashion.
+
+# Problem
+
+A team is open to receiving contributions from downstream users of their
+component. Coordination and communication happens in an ad-hoc fashion though
+leading to incoherent information being shared, delays in answers received,
+contributors pinging multiple host team members before receiving a definitive
+answer.
+
+# Forces
+
+- The host team is interested in receiving contributions and willing to mentor
+  contributors.
+- Teams have a strong verbal communication culture and are inexperienced with
+  setting up project specific asynchronous communication channels.
+- Communication channels may be aligned with specific groups that should be
+  reached but not by communication purpose.
+
+# Solution
+
+The host team needs to be clear on the benefit of providing company-public,
+archived, searchable, linkable communication channels that are free to subscribe
+to by anyone in the company:
+
+- Others lurking and reading lowers the barrier to get involved raising the
```

Comment:
These bullets feel like part of the **Resulting Context** (Benefits).

Reply:
I agree. Moved and merged.